brave/browser-laptop
# [WiP] Advertise community.brave.com

Auditors: @bsclifton, @alexwykoff

Test Plan:
1. Make sure the anchor link to the community (discourse) on README works
2. Make sure "Submit Feedback..." in the Menu opens the community in a new tab
3. Make sure "Send us feedback..." on about:preferences opens it in a new tab
